tests/utils.py

```diff
@@ -528,6 +528,12 @@ def __init__(self, *args, **kwargs):
         super(DummyTracer, self).__init__()
         self._trace_flush_enabled = True
         self.configure(*args, **kwargs)
+        if ddtrace.config._data_streams_enabled:
+            # Inline the import to avoid pulling in ddsketch or protobuf
+            # when importing ddtrace.
+            from ddtrace.internal.datastreams.processor import DataStreamsProcessor
+
+            self.data_streams_processor = DataStreamsProcessor(self._agent_url)
```

Comment:
I am not sure if this required. `super(DummyTracer, self).__init__()` should initialize the data stream process if  `ddtrace.config._data_streams_enabled=True`